ripd/ripd.h

```diff
@@ -539,4 +539,6 @@ DECLARE_HOOK(rip_ifaddr_del, (struct connected * ifc), (ifc));
 
 extern void rip_ecmp_change(struct rip *rip);
 
+extern uint32_t zebra_ecmp_count;
```

Comment:
shouldn't we set zebra_ecmp_count in the rip_zebra.c capabilities function?

Reply:
rip_nb_config.c does not include rip_zebra.c or how do you suggest doing this?